rooms/views.py

```diff
@@ -223,3 +286,350 @@ def get(self, request, room_id):
+                            
+        result = [
+            {
+                'room_id'       : wish.room.id,
+                'price'         : wish.room.price,
+                'point_average' : FOMATTER(ReviewEvaluation.objects.filter(
+                                    review_id__in = wish.room.roomreview_set.values('id')
+                                ).aggregate(Avg('point'))['point__avg']),
+                'user_wish'     : 1,
+                'room_name'     : wish.room.name,
+                'room_image'    : [image.s3_url for image in wish.room.image_set.all()],
+                'review_count'  : wish.room.roomreview_set.all().aggregate(Count('id'))['id__count']
+            } for wish in wishes
+        ]
+
+        return JsonResponse({'wishes' : result}, status=200)
+
+    ### 위시 리스트 담기 ###
+    @login_required
+    def post(self, request):
+        try:
+            user_id = request.user
+            data    = json.loads(request.body)
+            room_id = data['room_id']
+
+            if not room_id:
+                return JsonResponse({'MESSAGE':'INVALID REQUEST'}, status=400)
+
+            obj, created = Wish.objects.get_or_create(
+                user_id = user_id,
+                room_id = data['room_id']
+            )
+
+            if not created:
+                return JsonResponse({'MESSAGE' : 'EXISTS WISH ROOM'}, status=400)
+            
+            return JsonResponse({'MESSAGE' : 'SUCCESS'}, status=201)
+
+        except KeyError:
+            return JsonResponse({'MESSAGE' : 'INVALID REQUEST'}, status=400)
+        except json.decoder.JSONDecodeError:
+            return JsonResponse({'MESSAGE' : 'EMPTY BODY'}, status=400)
+
+    ### 위시리스트 삭제 ###
+    @login_required
+    def delete(self, request):
+        try:
+            user_id = request.user
+            room_id = int(request.GET.get('room_id', 0))
+
+            is_deleted = Wish.objects.filter(user_id = user_id, room_id = room_id).delete()
+
+            if not is_deleted[0]:
+                return JsonResponse({'MESSAGE' : 'NOT EXISTS WISH ROOM'}, status=400)
+
+            return JsonResponse({'MESSAGE' : 'SUCCESS'}, status=204)
+
+        except ValueError:
+            return JsonResponse({'MESSAGE' : 'INVALID REQUEST'}, status=400)
+
+class Order(View):
+    ### 방예약 ###
+    @login_required
+    def post(self, request):
+        try:
+            user_id = request.user
+
+            datas = json.loads(request.body)
+
+            check_in_date  = datas['check_in_date']
+            check_out_date = datas['check_out_date']
+            adult          = datas['adult']
+            kids           = datas['kids']
+            baby           = datas['baby']
+            room_id        = datas['room_id']
+
+            if not(check_in_date and check_out_date and 
+                    adult and kids and baby and room_id):
+                return JsonResponse({'MESSAGE' : 'INVALID REQUEST'}, status=400)
```

Comment:
아래와 같이 더욱 하드하게 input을 검증할 수 있는 방법이 있습니다~

```suggestion
            if not isinstance(check_in_date, date):
                ..
            if not isinstance(check_out_date, date):
                ..
            if not isinstance(room_id, int):
                ...

```